src/export/exporter.rs

```diff
@@ -129,6 +134,11 @@ impl Exporter {
         self.native.as_ref()
     }
 
+    /// Get the seconds to limit processing to.
+    pub fn processing_limit(&self) -> Option<f32> {
```

Comment:
We might be able to improve this name, too. `processing_limit` doesn't specify either what kind of processing we're doing, or the units we use to measure the limit. Maybe `end_time`? And then we could change `--limit` to `--end-time`, perhaps? I'm open to other suggestions, of course!